<?xml version="1.0"?>
<launch>
  <arg name="bag" default="" />
  <!-- Line Detection Node -->
  <node name="line_detection" pkg="planning_ros" type="line_detection_node" output="screen">
    <rosparam command="load" file="$(find planning_ros)/config/line_detection.yaml" />
  </node>
  <node pkg="rosbag" type="play" name="rosbag_play" output="screen" args="$(arg bag)" if="$(eval arg('bag') != '')" />
</launch>
